katello/hooks/post/30-upgrade.rb

```diff
@@ -12,7 +17,11 @@ def fail_and_exit(message)
 
 if app_value(:upgrade)
   if [0, 2].include?(@kafo.exit_code)
-    upgrade_tasks if module_enabled?('foreman')
+    if module_enabled?('foreman')
+      db_seed
```

Comment:
Should we run `db:migrate` to be sure as well? The katello package doesn't run this in post so you're not sure it ran. Since it's idempotent, it should be safe.

Reply:
I think it's a good idea!

Reply:
Actually, looks like it's done here:

https://github.com/theforeman/foreman-installer/blob/220b749e223b481761afb32559b56e3a75ce59ea/katello/hooks/pre/30-upgrade.rb#L226

Reply:
It feels more correct like a post, but let's keep that out of this PR so we can easily (and safely) cherry pick this.